extensions/wikia/PhalanxII/PhalanxStatsSpecialController.class.php

```diff
@@ -12,45 +12,124 @@ function __construct( ) {
 	}
 
 	public function index() {
-		wfProfileIn( __METHOD__ );
 		$this->wg->Out->setPageTitle( wfMsg( 'phalanx-stats-title' ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "phalanx-stats-title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)